### PR TITLE
Fix/search bar link

### PR DESCRIPTION
**Pull request by ClaudivanFilho**
#### What is the purpose of this pull request?
- Internationalization to `SearchBar`
- Spinner to `SearchBar`
- `SearchBar` options link 

#### Screenshots or example usage
[workspace link](https://claudivan--storecomponents.myvtex.com/)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SearchBar/components/AutocompleteInput.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import Input from '@vtex/styleguide/lib/Input'
 export default class AutocompleteInput extends Component {
   render() {
     return (
-      <Input size="large" onKeyPress={this.props.onKeyDown} {...this.props} />
+      <Input size="large" onKeyPress={(evt) => {
```
Comment (andreldsa) [inline react/components/SearchBar/components/AutocompleteInput.js]: event*

File: react/components/SearchBar/components/ResultsList.js [hunk 2]
```diff
@@ -37,41 +61,34 @@ class ResultsList extends Component {
...
-                    src={getImageUrl(el.thumb)}
-                  />
-                </div>
-              )}
-              <div className="flex justify-center items-center">{el.name}</div>
-            </Link>
           </li>
-        ))}
+        </Link>
+        {items.map((el, index) => {
```
Comment (andreldsa) [inline react/components/SearchBar/components/ResultsList.js]: el? Better variable

File: react/components/SearchBar/index.js [hunk 3]
```diff
@@ -1,54 +1,66 @@
...
               selectedItem,
               highlightedIndex,
               isOpen,
+              closeMenu
             }) => (
               <div className="relative">
                 <AutocompleteInput
                   {...getInputProps({ placeholder })}
-                  onKeyDown={this.handleEnterPress}
+                  onKeyDown={evt => {
```
Comment (andreldsa) [inline react/components/SearchBar/index.js]: event

File: react/components/SearchBar/components/AutocompleteInput.js [hunk 4]
```diff
@@ -7,7 +7,9 @@ import Input from '@vtex/styleguide/lib/Input'
 export default class AutocompleteInput extends Component {
   render() {
     return (
-      <Input size="large" onKeyPress={this.props.onKeyDown} {...this.props} />
+      <Input size="large" onKeyPress={(event) => {
```
Comment (brunojdo) [inline react/components/SearchBar/components/AutocompleteInput.js]: Use `vtex.styleguide`

**Pull request merged**